analyses/maven/matcher_test.go

```diff
@@ -0,0 +1,85 @@
+package maven_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/fabric8-analytics/cli-tools/analyses/driver"
+	"github.com/fabric8-analytics/cli-tools/analyses/maven"
+)
+
+type isSupportedManifestTestcase struct {
+	Name    string
+	Want    bool
+	Matcher driver.StackAnalysisInterface
+}
+
+type depsTreeTestCase struct {
+	ManifestFile string
+	Want         string
+	Matcher      driver.StackAnalysisInterface
+}
+
+func (tc isSupportedManifestTestcase) TestMatcher_IsSupportedManifest(t *testing.T) {
+	_, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	got := tc.Matcher.IsSupportedManifestFormat(tc.Name)
+	want := tc.Want
+	if got != want {
+		t.Errorf("got: %v, want: %v", got, want)
+	}
+}
+
+func (dt depsTreeTestCase) TestDepsTree_GeneratorDependencyTree(t *testing.T) {
+	_, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
Context is unused here. Actually it should be passed over to the functions if you need to make use of it.

Applicable for All tests

Reply:
Ack.